--- xfce4-clipman-plugin-1.2.3/doc/Makefile.am~	2011-10-29 08:26:20.000000000 +0200
+++ xfce4-clipman-plugin-1.2.3/doc/Makefile.am	2012-05-17 21:09:43.725977965 +0200
@@ -4,7 +4,7 @@
 MALLARD_CSS = index.css
 MALLARD_XSL = index.xsl
 
-TARGET_DIR = $(DESTDIR)$(docdir)/html
+TARGET_DIR = $(DESTDIR)$(datadir)/xfce4/doc
 LINGUAS = $(shell cd $(srcdir)/po && echo `/bin/ls *.po 2> /dev/null | sed 's,[.]po,,g'`)
 
 EXTRA_DIST = $(MALLARD_PAGE) $(MALLARD_CSS) $(MALLARD_XSL)
